spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
```

Comment:
- [ ] It would be better if we change the name of the `spec/controllers` to `spec/requests` as seen in the requirements below

![Screenshot from 2022-10-26 11-59-01](https://user-images.githubusercontent.com/46388113/198020744-77efe1eb-e2e7-473b-b4da-e49e5d39e904.png)